src/main/java/org/opentripplanner/routing/vehicle_parking/VehicleParkingGroup.java

```diff
@@ -0,0 +1,86 @@
+package org.opentripplanner.routing.vehicle_parking;
+
+import java.util.Locale;
+import java.util.Objects;
+import javax.annotation.Nullable;
+import org.opentripplanner.transit.model.basic.I18NString;
+import org.opentripplanner.transit.model.basic.WgsCoordinate;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+/**
+ * Group of vehicle parking locations.
+ * <p>
+ * All fields are immutable. If any other properties change a new VehicleParkingGroup instance
+ * should be created.
+ */
+public class VehicleParkingGroup {
+
+  private final FeedScopedId id;
+
+  private final I18NString name;
+
+  private final WgsCoordinate coordinate;
+
+  VehicleParkingGroup(VehicleParkingGroupBuilder vehicleParkingGroupBuilder) {
+    this.id = vehicleParkingGroupBuilder.id;
+    this.name = vehicleParkingGroupBuilder.name;
+    this.coordinate = vehicleParkingGroupBuilder.coordinate;
+  }
+
+  public static VehicleParkingGroupBuilder builder() {
+    return new VehicleParkingGroupBuilder();
+  }
```

Comment:
you could name this `of` and add the id as the parameter, similar to how the transit entities are built